asmcomp/cmmgen.ml

```diff
@@ -1262,7 +1262,7 @@ and transl_prim_4 env p arg1 arg2 arg3 arg4 dbg =
   | Pbigarrayref (_, _, _, _) | Pbigarrayset (_, _, _, _) | Pbigarraydim _
   | Pstring_load _ | Pbytes_load _ | Pbigstring_load _ | Pbbswap _
     ->
-      fatal_errorf "Cmmgen.transl_prim_3: %a"
+      fatal_errorf "Cmmgen.transl_prim_4: %a"
```

Comment:
This is an unrelated typo in the error message (from https://github.com/ocaml/ocaml/pull/12735). This was fixed while rebasing.